src/mime-node.js

```diff
@@ -197,11 +197,10 @@ export default class MimeNode {
     }
 
     processHeaders() {
-        for (let i = this.headerLines.length - 1; i >= 0; i--) {
+        for (let i = 0; i <= this.headerLines.length - 1; i++) {
             let line = this.headerLines[i];
             if (i && /^\s/.test(line)) {
-                this.headerLines[i - 1] += '\n' + line;
-                this.headerLines.splice(i, 1);
+                continue;
```

Comment:
I didn't quite grasp what these 2 lines were for (appending a line of whitespace to the line prior to it?) and wasn't sure how to preserve the behavior easily with the loop's reversed direction

Reply:
I think I better understand what this was for now after running into an issue without it in the app-side fix I've been trying to make.

Or at least the issue I was hitting was when a line preceeding the Content-Type header starts with whitespace, then nothing gets processed correctly.

So this is actually collapsing lines that start with whitespace onto previous lines so that no lines start with whitespace.

I'll go ahead and split this into 2 loops to do this, so only the 1st one has to operate in reverse direction